docs/components/props/Proptable.tsx

```diff
@@ -0,0 +1,136 @@
+import getConfig from "next/config";
+import { useState } from "react";
+import "./proptable.css";
+import Inline from "../code/Inline";
+import "nav-frontend-tabell-style";
+import { Select } from "nav-frontend-skjema";
+import Import from "../code/Import";
+
+const Proptable = ({
+  names,
+  defaultExport,
+  from,
+}: {
+  names: string[];
+  defaultExport?: string;
+  from: string;
+}) => {
+  const [tab, setTab] = useState(0);
+  const { publicRuntimeConfig } = getConfig();
+
+  const props = publicRuntimeConfig.props
+    .filter((comp) => [...names, defaultExport].includes(comp.displayName))
+    .map((comp) => {
+      if (comp.displayName === defaultExport) {
+        return { ...comp, defaultExport: true };
+      }
+      return { ...comp, defaultExport: false };
+    })
+    .sort((a, b) => a.defaultValue === b.defaultValue);
+
+  const headers = ["Prop", "Type", "Description"];
+
+  const ProptableHeader = (props) => <th key={props.index}>{props.val}</th>;
+
+  const parsePropValue = (val) => {
+    if (val && typeof val === "object") {
+      if (val.name === "enum") {
+        return val.value.map((x) => x.value).join(" | ");
+      }
+      val = val.name || val.value; // eslint-disable-line no-param-reassign
+    }
+    if (val === null || typeof val === "undefined" || val.length <= 0) {
+      return "-";
+    }
+    return val.toString();
+  };
+
+  const parseDescription = (desc) => {
+    const found = desc.match(/^@deprecated/);
+    if (found) {
+      return (
+        <span>
+          <strong>{found[0]}</strong>
+          {desc.substr(11, desc.length)}
+        </span>
+      );
+    }
+    return desc;
+  };
+
+  const PropTypeTableRow = (props) => {
+    const desc = parseDescription(parsePropValue(props.val.description));
+    return (
+      <tr className="proptable__tr">
+        <td>
+          <div>
+            <strong>{parsePropValue(props.val.name)}</strong>
+          </div>
+        </td>
+        <td>
+          <div>
+            <Inline>{parsePropValue(props.val.type)}</Inline>
+          </div>
+        </td>
+        {/* <td>
+          <div>{parsePropValue(props.val.defaultValue)}</div>
```

Comment:
Trenger vi å ta vare på denne koden?

Reply:
Proptable er enda litt WIP og er litt usikker på hva vi ønsker å vise..
Eks `prop, type, desc, default, required`? Eller bare `prop, type, desc` som nå?

Reply:
Jeg stemmer for å ha med alle de.

Reply:
Siden fem table columns tar så mye plass kan vi kanskje ha en lignende løsning som polaris her? https://polaris.shopify.com/components/actions/account-connection#prop-accountName

Alternativ til proptable her (en av disse):  
<img width="643" alt="Screenshot 2021-04-21 at 16 48 24" src="https://user-images.githubusercontent.com/26967723/115573894-6d3c7a80-a2c1-11eb-9576-08be70eeaf83.png">
<img width="638" alt="Screenshot 2021-04-21 at 16 47 34" src="https://user-images.githubusercontent.com/26967723/115573899-6e6da780-a2c1-11eb-925a-748a63ed7536.png">
Default kommer da opp under "required" her hvis den ikke er tom.